web/app/themes/dp-timber-theme/components/style.scss

```diff
@@ -1 +1,9 @@
-// Add global styling here.
\ No newline at end of file
+// Add global styling here.
+.sb-show-main.sb-main-padded {
```

Comment:
this seems specific to the storybook can we create a separate file specific to storybook overrides?